testdata/p4_16_bmv_errors/issue363-err-bmv2.p4

```diff
@@ -0,0 +1,101 @@
+#include <v1model.p4>
+
+struct alt_t {
+  bit<1> valid;
+  bit<7> port;
+};
+
+struct row_t {
+  alt_t alt0;
+  alt_t alt1;
+};
+
+struct local_metadata_t {
+  row_t row;
+};
+
+struct mac_t
+{
+    bit<28>   lower28Bits;
+    bit<20>   upper20Bits;
+}
+
+struct macDA_t
+{
+    mac_t     macDA;
+}
+
+header bitvec_hdr {
+    bit<5> g2;
+    bit<3> g3;
+    macDA_t mac;
+}
+
+struct parsed_packet_t {
+    bitvec_hdr bvh;
+    bitvec_hdr bvh1;    
+};
+
+struct tst_t {
+    bitvec_hdr bvh0;
+}
+
+parser parse(packet_in pk, out parsed_packet_t h,
+             inout local_metadata_t local_metadata,
+             inout standard_metadata_t standard_metadata) {
+  state start {
+    pk.extract(h.bvh);
+    pk.extract(h.bvh1);
+    transition accept;
+  }
+}
+
+control ingress(inout parsed_packet_t h,
+                inout local_metadata_t local_metadata,
+	        inout standard_metadata_t standard_metadata) {
+    apply {
+        tst_t s;
+
+        bit<48> mac;
+        // h.bvh.mac.macDA.lower28Bits = 0;
```

Comment:
please delete commented-out code if you don't want it.
I would make this negative test case as small as necessary; it should only test the things that are supposed to fail, and nothing else.

Reply:
Done.  Now the code only includes and error to show Cast between bit-fields and bit-vector struct of same width is not supported.